tests/computer_player_test.rb

```diff
@@ -0,0 +1,59 @@
+require 'minitest/autorun'
+require './lib/ship'
+require './lib/cell'
+require './lib/board'
+require './lib/computer_player'
+
+class ComputerPlayerTest < Minitest::Test
+
+  def setup
+    @comp_board = Board.new#(@dim)
+    @player_board = Board.new
+    @cruiser = Ship.new("Cruiser", 3)
+    @submarine = Ship.new("Submarine", 2)
+    @comp_player = ComputerPlayer.new(@player_board,
+                                      @comp_board,
+                                      [@cruiser, @submarine])
+    @player_board.place(Ship.new("test",3), ["A1","A2","A3"])
+
+  end
+
+  def test_place_own_ships_places_all_ships
+    @comp_player.place_own_ships
+
+    ships_placed = []
+
+    ships_placed = @comp_board.cells.keys.find_all{
+      |coordinate|
+      @comp_board.cells[coordinate].ship != nil}
+
+    ships_placed = ships_placed.map{
+      |coordinate|
+      @comp_board.cells[coordinate].ship.name}
+    p ships_placed
+    assert_equal 2, ships_placed.uniq.length
+  end
+
+  def test_placed_ships_occupy_correct_number_cells
```

Comment:
Nice, this test will scale too for ships of different length and more than 2 ships